core/consensus/grandpa/impl/environment_impl.cpp

```diff
@@ -225,6 +225,11 @@ namespace kagome::consensus::grandpa {
     return outcome::success();
   }
 
+  bool EnvironmentImpl::contains(
```

Comment:
Since Environment is a pretty generic interface, I think that it'd be better to explicitly name it containsBlock or something alike.